helpers/clean-kibana-indexes.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+# Description: script to remove old .kibana* indexes' aliases created after Kibana
+# restart.
+#
+# The aim of the script is to keep only the latest index with the latest 
+# configuration, and delete the old ones.
+
+# Usage is: 
+# CLUSTER="amazeeio-test" ELASTICSEARCH_NAMESPACE="elasticsearch" ./clean-kibana-indexes.sh"
+# CLUSTER="amazeeio-test" ELASTICSEARCH_NAMESPACE="elasticsearch" TENANT="drupalexample" ./clean-kibana-indexes.sh"
+
+set -eu
+
+# Set DEBUG variable to true, to start bash in debug mode
+DEBUG="${DEBUG:-"false"}"
+if [ "$DEBUG" = "true" ]; then
+        set -x
+fi
+
+# Cluster to connect to
+CLUSTER="${CLUSTER:-""}"
+if [ -z "$CLUSTER" ]; then
+	echo -e "Set CLUSTER variable\n"
+	echo -e "Usage is: CLUSTER="amazeeio-test" ELASTICSEARCH_NAMESPACE="elasticsearch" ./clean-kibana-indexes.sh"
+	exit 1
+fi
+
+# Set the ACTION of the script between CLOSE or DELETE
+ACTION="${ACTION:-""}"
+if [ -z "$ACTION" ]; then
+	echo -e "Set ACTION variable to CLOSE or DELETE\n"
+	echo -e "Usage is: CLUSTER="amazeeio-test" ELASTICSEARCH_NAMESPACE="elasticsearch" ACTION=CLOSE ./clean-kibana-indexes.sh"
+	exit 1
+fi
+
+# Set DRYRUN option to "true" to run in a dry-run mode
+DRYRUN="${DRYRUN:-"false"}"
+
+# Optional set a tenant to check and close/delete indexes
+TENANT="${TENANT:-"*"}"
+if [ "$TENANT" = "*" ]; then
+	REGEX="*"
```

Comment:
this is not a regex, let's call this what this actually is

Reply:
I called INDEXES